docs/ReverseProxyToTor.md

```diff
@@ -215,6 +215,127 @@ systemctl restart nginx
 
 Now, visiting `mydomain.com` should show your BTCPay Server instance.
 
+
+## Do all this in a Docker container
+
+Ready made docker image 
+
+  https://hub.docker.com/r/cloudgenius/socator
+
+Code behing the image
+
+  https://github.com/beacloudgenius/socator
+
+```
+### SocaTor = SOCAT + TOR
+Based on https://github.com/Arno0x/Docker-Socator
+
+It uses socat to listen on a given TCP port (5000 in this example) and to redirect incoming traffic to a tor hidden service specified through environment variables. Itacts as a relay between the standard web and a hidden service on the tor network. You can optionally restrict the IP addresses that are allowed to connect to this service by specifying an `ALLOWED_RANGE` environment variable and using CIDR notation.
+
+### Usage
+
+Break free from cloud services providers limitations, secure and protect your bitcoin full node, connect that with a BTC Pay server, all behind TOR.
+Selectively expose the BTC Pay Server payment gateway and API to clearnet using socat+tor running on the Internet.
+
+--------------
+
+##### build
+
+    docker build -t cloudgenius/socator .
+
+##### push
+
+    docker push cloudgenius/socator
+
+##### Start the image in background (*daemon mode*) with IP address restriction:
+
+    docker run -d \
+        -p 5000:5000 \
+        -e "ALLOWED_RANGE=192.168.1.0/24" \
+        -e "TOR_SITE=zqktlwiuavvvqqt4ybvgvi7tyo4hjl5xgfuvpdf6otjiycgwqbym2qad.onion" \
+        -e "TOR_SITE_PORT=80" \
+        --name socator \
+        cloudgenius/socator
+
+##### Start the image in foreground:
+
+    docker run --rm -ti \
+        -p 5000:5000 \
+        -e "TOR_SITE=zqktlwiuavvvqqt4ybvgvi7tyo4hjl5xgfuvpdf6otjiycgwqbym2qad.onion" \
+        -e "TOR_SITE_PORT=80" \
+        --name socator \
+        cloudgenius/socato
+
+Now http://localhost:5000 should show you the tor hidden service you specified in the above command.
+
+```
+
+
+## Use that Docker container in a Kubernetes Cluster using these manifests
```

Comment:
Does BTCPay Server run inside Kubernetes? Didn't know we support that?

Reply:
Hehe, me neither :)

@lvnilesh Can you use the code fence notation for the code samples as we do in the rest of the docs?

Reply:
I find it best for privacy and security to run a full node, NBxplorer, and BTCpayserver all behind TOR and completely in my physical possession and full control.

However, I still want to expose the payment endpoint to the public clearnet without asking BTC payers to go through TOR. 

Enter socator container running in kubernetes.

In my example, I am running socator container in kubernetes in the cloud and everything else runs in my control and supervision.

I will examine the code fence notation next and update the PR.

Reply:
code fence notation updated to match the rest.